pages/agent/v3/windows.md.erb

```diff
@@ -4,42 +4,36 @@ The Buildkite Agent is supported on Windows 8, Windows Server 2012, and newer.
 
 {:toc}
 
-## Automated install with Powershell
+## Automated install with PowerShell
 
-You'll need to run the automated installer within Powershell with Administrative Privileges.
+You'll need to run the automated installer within PowerShell with administrative privileges.
 
-To start a Powershell session as an Administrator:
+Once you're in an escalated PowerShell session, you can run this script to install the latest version of the agent:
 
-1. Open the Start menu and type "powershell"
-2. Highlight "Windows Powershell" in the results
-3. Press CTRL+SHIFT+ENTER (you'll be prompted to allow this app to make changes)
-
-Once you're in an escalated Powershell session, you can run this script to install the latest version of the Agent:
-
-```
-$env:buildkiteAgentToken = "my_token"
+```shell
+$env:buildkiteAgentToken = "<your_token>"
```

Comment:
Does Windows have a typical command prompt? And would it be useful to include it?  I can't tell how many commands this is.

Reply:
Generally no. Looking at PowerShell's own docs, they're a bit inconsistent themselves. Some have nothing, some have an example path, and some have PS>. However PS> will make no sense for someone using command prompt. And > will make no sense for someone using Git Bash, which Buildkite encourages.

But this is a PowerShell specific example, so I've gone with PS>